bin/i18n/sync-in.rb

```diff
@@ -297,4 +298,12 @@ def redact_block_content
   RedactRestoreUtils.redact(source, source, ['blockfield'], 'txt')
 end
 
+def sanitize_markdown_headers
```

Comment:
We've talked in the past about cleaning up the way we pull in Markdown files (ie moving out of the in.sh script). Now that we have to explicitly touch the markdown files in the in step I could move that logic here. Thoughts?

Reply:
I'm all about that!